### PR TITLE
Update copyright to 2021 and khal contributors

### PR DESCRIPTION
**Pull request by geier**

### CODE REVIEW

**Comment by d7415** [comment]
Looks good. Did you want to change [doc/source/conf.py](https://github.com/pimutils/khal/blob/master/doc/source/conf.py) too?

I also found https://github.com/pimutils/khal/blob/master/setup.py#L41-L42 - I don't know whether you want to remove your email from there too?

**Comment by geier** [comment]
> Looks good. Did you want to change [doc/source/conf.py](https://github.com/pimutils/khal/blob/master/doc/source/conf.py) too?
> 
> I also found https://github.com/pimutils/khal/blob/master/setup.py#L41-L42 - I don't know whether you want to remove your email from there too?

Thanks, I definitively missed those.

I left my email and original website in there for lack of better alternative. We can probably create a new @pimutils.org address and forward to all maintainers.

**Comment by d7415** [comment]
> I left my email and original website in there for lack of better alternative. We can probably create a new @pimutils.org address and forward to all maintainers.

Whichever you're happier with. I just thought I'd mention it as you'd removed it as a support contact elsewhere.

File: doc/source/conf.py [hunk 1]
```diff
@@ -330,7 +330,7 @@ def write_section(specsection, secname, key, comment, output):
 #  dir menu entry, description, category)
 texinfo_documents = [
     ('index', 'khal', 'khal Documentation',
-     'Christan Geier et al.', 'khal', 'One line description of project.',
+     'khal contributors', 'khal', 'A standards based calendar program',
```
Comment (d7415) [inline doc/source/conf.py:333]: Ooh, good spot!

**Review by d7415 — APPROVED** [review]

**Pull request merged**